app/services/catalog/create_approval_request.rb

```diff
@@ -0,0 +1,66 @@
+require 'approval_api_client'
+
+module Catalog
+  class CreateApprovalRequest
+    attr_reader :order
+
+    def initialize(id)
+      @order = Order.find_by!(:id => id)
+
+      ApprovalApiClient.configure do |config|
+        config.api_key['x-rh-auth-identity'] = ManageIQ::API::Common::Request.current.request_id
```

Comment:
@lindgrenj6 interested that the Approval service is using `x-rh-auth-identity`.  The IPP notes this should be called `x-rh-identity`.

Reply:
Follow the IPP.  Sounds like approval needs to be updated cc @bzwei @hsong-rh 

Also, this client setup should move to lib so it can be reused.

Reply:
@gmcculloug https://github.com/ManageIQ/approval-api/pull/69 is created to correct the key name.